Base/usr/share/man/man7/Tips-and-Tricks.md

```diff
@@ -0,0 +1,63 @@
+![Welcome Icon](/res/icons/32x32/app-welcome.png)
+
+# Tips and Tricks
+
+This is a list of useful tips and tricks to help you make the most out of SerenityOS
+
+Related: [Keyboard Shortcuts](help://man/7/KeyboardShortcuts)
+
+## General
```

Comment:
All of the lists in this file could do with some periods at the end of most items.